week1/Week1/src/week1/Average2.java

```diff
@@ -0,0 +1,30 @@
+
+package week1;
+
+import java.util.Scanner;
+
+/**
+ *
+ * @author sara
+ */
+public class Average2 {
+
+  
+    public static void main(String[] args) {
+      double grade1,grade2,grade3;
+      Scanner input=new Scanner(System.in);
+      grade1=input.nextDouble();
+      grade2=input.nextDouble();
+      grade3=input.nextDouble();
```

Comment:
Use white spaces to make it more readable and easier to understand.
ex: grade1  = input.nextDouble();